pkg/handler/handler.go

```diff
@@ -94,33 +94,34 @@ func createVarHandler(r VarHandler) GorillaRouteHandler {
 }
 
 // NewHandler - Create a new handler by attaching the routes and setting logger and broker.
-func NewHandler(b broker.Broker, log *logging.Logger, brokerConfig broker.Config) http.Handler {
+func NewHandler(b broker.Broker, log *logging.Logger, brokerConfig broker.Config, prefix string) http.Handler {
 	h := handler{
 		router:       *mux.NewRouter(),
 		broker:       b,
 		log:          log,
 		brokerConfig: brokerConfig,
 	}
+	s := h.router.PathPrefix(prefix).Subrouter()
 
 	// TODO: Reintroduce router restriction based on API version when settled upstream
```

Comment:
This covers the TODO here.

Reply:
can you explain?

Reply:
I thought 104 covered this todo. But, I think I'm confusing a Headers and a pathprefix. They aren't related.